argoverse/utils/make_att_files.py

```diff
@@ -304,27 +306,22 @@ def compute_v_a(traj: np.array) -> Tuple[np.array, np.array]:  # traj:Nx3
                     ind_close_max = ind_close.max() + 1
                     ind_close_min = ind_close.min()
 
-                if dict_tracks[id_track]["length_track"] == 0:
-                    # this shouldn't happen
-                    dict_tracks[id_track]["difficult_att"].append("zero_length")
-
+                if dict_tracks[id_track]["list_dist"][ind_valid].min() > dist_close:
+                    dict_tracks[id_track]["difficult_att"].append("far")
                 else:
-                    if dict_tracks[id_track]["list_dist"][ind_valid].min() > dist_close:
-                        dict_tracks[id_track]["difficult_att"].append("far")
+                    if dict_tracks[id_track]["length_track"] < 10 or dict_tracks[id_track]["exists"].sum() < 10:
+                        dict_tracks[id_track]["difficult_att"].append("short")
                     else:
-                        if dict_tracks[id_track]["length_track"] < 10 or dict_tracks[id_track]["exists"].sum() < 10:
-                            dict_tracks[id_track]["difficult_att"].append("short")
-                        else:
-                            if (ind_close_max - ind_close_min) - dict_tracks[id_track]["exists"][
-                                ind_close_min:ind_close_max
-                            ].sum() > 20:
-                                dict_tracks[id_track]["difficult_att"].append("occ")
+                        if (ind_close_max - ind_close_min) - dict_tracks[id_track]["exists"][
+                            ind_close_min:ind_close_max
+                        ].sum() > 20:
```

Comment:
can we make 20 a module-level private variable? e.g. `MAX_OCCLUSION_PCT_ = 20`

Reply:
ok